<launch>

    <node name="vehicle_mission" pkg="vehicle_mission" type="detect_lane.py" output="screen">
        <param name="kernel" value="10"/>
        <param name="threshold_low" value="200"/>
        <param name="threshold_high" value="255"/>
    </node>

</launch>